### ISSUE TITLE
数据的处理和输入无法执行

### ISSUE THREAD

**Issue by currtlly** [zhaoqi106/DCAMCP#1]
数据转tensor数据类型时会报错，字符串无法转为tensor怎么处理？例如：fps = torch.tensor(z_train)，return batched_graph, torch.tensor(labels), torch.tensor(test_list_fp)。请问一下作者是少发了一部分数据处理代码吗？
